packages/host/app/services/matrix-service.ts

```diff
@@ -250,20 +237,24 @@ export default class MatrixService extends Service {
   public async createRealmSession(realmURL: URL) {
     await this.ready;
     if (!this.isLoggedIn) {
-      throw new Error(
+      console.warn(
```

Comment:
Why change to warn? Should this not be a hard error?

Reply:
reverted it here [060e44d](https://github.com/cardstack/boxel/pull/1020/commits/060e44db588f70f88c23ba4e728012099335f55f).